schism/compiler.ss

```diff
@@ -133,6 +134,25 @@
              ;; Add one dword to the pointer we allocated to make sure
              ;; we skip the base air.
              (%set-tag (+ allocation 1) tag))))
+       (define (make-string n)
+         ;; Allocate n + 1 words. %alloc will automatically round up
+         ;; if needed.
+         (let* ((num-words (+ 1 n))
+                (s (%alloc ,(string-tag) num-words)))
```

Comment:
Not initializing the characters seems a bit dangerous to me, given that they are represented as tagged values.

Reply:
You're right, although at the moment this should be safe. The reason is that Wasm zero-initializes its memory, and we never reuse memory at the moment. That said, at some point we probably will reuse memory and this will come back to bite us.

I'll probably split it into `unsafe-make-string` that doesn't initialize and `make-string` which does, so we can avoid the initialization cost for the compiler-generated sequences that are immediately followed by a bunch of `string-set!` calls.

Reply:
Makes sense to me.  Note that after a GC, afaiu, we would be re-using old memory I think.